src/ast/CMakeLists.txt

```diff
@@ -11,8 +11,12 @@ target_include_directories(ast PUBLIC ${CMAKE_SOURCE_DIR}/src/ast)
 target_include_directories(ast PUBLIC ${CMAKE_BINARY_DIR})
 target_link_libraries(ast arch)
 
-add_dependencies(ast bcc-build parser)
-ExternalProject_Get_Property(bcc source_dir)
+if (SYSTEM_BCC_LIBRARY)
+  add_dependencies(ast parser)
+else()
+  add_dependencies(ast bcc-build parser)
+  ExternalProject_Get_Property(bcc source_dir)
+endif()
 target_include_directories(ast PUBLIC ${source_dir}/src/cc)
```

Comment:
We should move this into the else block above too (or remove if we're going to go entirely with `include_directories()`)

Reply:
Fixed